turbo/jsonrpc/debug_api.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/ledgerwatch/erigon/core/state"
 	"github.com/ledgerwatch/erigon/core/types/accounts"
 	"github.com/ledgerwatch/erigon/eth/stagedsync/stages"
-	"github.com/ledgerwatch/erigon/eth/tracers"
+	tracerConfig "github.com/ledgerwatch/erigon/eth/tracers/config"
```

Comment:
create separated 1 separated PR for such changes. easy to merge. it will help to reduce size of this PR. tnx.

Reply:
https://github.com/ledgerwatch/erigon/pull/10782